common/thorhelper/thormeta.cpp

```diff
@@ -59,36 +59,41 @@ CLogicalFile::CLogicalFile(const CStorageSystems & storage, const IPropertyTree
     fileSize = xml->getPropInt64("@rawSize"); // logical file size - used for file positions
```

Comment:
'xml' is a bit misleading/confusing (led me to think it was something to do with XML reading at 1st glance)
can it be renamed 'meta' or 'fileMeta'?